pkg/repo/repotest/server.go

```diff
@@ -155,6 +155,7 @@ func (srv *OCIServer) Run(t *testing.T, opts ...OCIServerOpt) {
 		ociRegistry.ClientOptEnableCache(true),
 		ociRegistry.ClientOptWriter(os.Stdout),
 		ociRegistry.ClientOptCredentialsFile(credentialsFile),
+		ociRegistry.ClientOptPlainHTTP(),
```

Comment:
didn't see a way to enable OCI TLS easily but doesn't seem needed in the other tests